README.md

```diff
@@ -99,6 +99,30 @@ fastify.after(() => {
 })
 ```
 
+### How to add credentials to a request
+The basic-auth plugin retrieves the credentials from the [Authorization](https://en.wikipedia.org/wiki/Basic_access_authentication) header.
+
+A simple example to add `username` and `password` to the reply header:
```

Comment:
Why would we give an example showing sending the credentials in a response header? https://www.rfc-editor.org/rfc/rfc7617.html does not call for this, and it seems like a bad practice to recommend.